ash/src/entry.rs

```diff
@@ -54,33 +62,28 @@ pub enum InstanceError {
 
 impl fmt::Display for InstanceError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "InstanceError::{:?}", self)
-    }
-}
-
-impl Error for InstanceError {
-    fn description(&self) -> &str {
-        "InstanceError"
-    }
-
-    fn cause(&self) -> Option<&Error> {
-        if let &InstanceError::VkError(ref err) = self {
-            return err.source();
+        match self {
+            InstanceError::LoadError(e) => write!(f, "{}", e.join("\"n")),
```

Comment:
This is a strange delimiter. Perhaps you meant `"\n"`? I'd prefer `": "`, personally; sticking newlines in a `Display` impl tends to break peoples' formatting.

Reply:
Good eye! This is fixed